promql/lex.go

```diff
@@ -343,13 +343,6 @@ func (l *Lexer) run() {
 	}
 }
 
-// Release resources used by lexer.
-func (l *Lexer) close() {
-	for range l.Items {
```

Comment:
This used to have an effect, when l.Items was a channel. Now it is a slice, so the loop doesn't do anything.